pkg/cmd/controller/unidling.go

```diff
@@ -4,6 +4,7 @@ import (
 	"time"
 
 	"k8s.io/client-go/dynamic"
+	eventsv1 "k8s.io/client-go/kubernetes/typed/events/v1"
```

Comment:
I'd make this `eventsclient` (or `eventsv1client` as in the other file). `eventsv1` implies `"k8s.io/api/events/v1"`